.githooks/pre-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+STANDARD="Drupal,DrupalPractice"
+BIN="./vendor/bin"
+
+echo "~ ~ RUNNING PRE-COMMIT HOOKS ~ ~"
+echo "RUNNING COMPOSER VALIDATE"
+composer validate
+
+echo "RUNNING phpcbf"
+vendor/bin/phpcbf --standard=$STANDARD src
```

Comment:
this command `phpcbf` changes files. thus it would not be ideal to run this as a git pre-commit hook..  
instead print a message saying if there are errors run this cmd to resolve the issues.

Reply:
Can we also remove this cmd from the pre-commit